Leo Vazquez y Nico Mozo/src/views/Search.vue

```diff
@@ -105,10 +118,25 @@ export default {
     },
     onSubmit () {
       this.searchResults = [];
-      axios.get(this.getSourceURL(this.form.source), { params: { keywords: this.form.keyword } })
-      .then(function({ data }){
-        this.searchResults = data;
-      }.bind(this));
+      let strikes = 0;
+      let interval = setInterval(() => {
```

Comment:
`setInterval` fair enough... and this guy got employed by a European company, such a wonderful world...